javascript/0080-remove-duplicates-from-sorted-array-ii.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * Linear
+ * https://leetcode.com/problems/remove-duplicates-from-sorted-array-ii/
+ * Time O(n) | Space O(1)
+ * @param {number[]} nums
+ * @return {number}
+ */
+var removeDuplicates = function(nums) {
+    let current = nums[0];
+    let sameElCount = 0;
+
+    for(let i = 0; i < nums.length; i++) {
+      if(current === nums[i]) {
+          sameElCount++;
+      } 
+      if(current !== nums[i]) {
+          current = nums[i];
+          sameElCount = 1;
+      }
+     if(sameElCount > 2) {
+         nums.splice(i,1);
+         i--;
+      }
+    }
+};
+
+/**
+ * Two pointer
+ * Time O(n) | Space O(1)
+ * @param {number[]} nums
+ * @return {number}
+ */
+var removeDuplicates2 = function(nums) {
+
+    let current = nums[0];
+    let sameElCount = 0;
+
+
+    for(let i = 0; i < nums.length; i++) {
+      if(current === nums[i]) {
+          sameElCount++;
+      } 
+      if(current !== nums[i]) {
+          current = nums[i];
+          sameElCount = 1;
+      }
+      if(sameElCount > 2) {
+          let left = i;
+          let right = i+1;
+          let count = 1;
+          while(nums[left] === nums[right]) {
+              count++;
+              right++;
+          }
+          nums.splice(left, count);
```

Comment:
Suggestion: This will make your Time O(N^2)

Reply:
Oh yeah. Missed that one. Well caught!

Reply:
I have resolved the conflict.  And I have updated the time complexity.

Reply:
Just a kind reminder that the conflict is resolved. And updated the time complexity as suggested.